web/src/styles/scss/password-meter.scss

```diff
@@ -0,0 +1,80 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+.password-with-meter {
+  // inputs in the design system have a max width, so cap our container at that
+  // width too, to prevent floats and 100% widths from going too big
+  max-width: $input-max-width;
+
+  .password-with-meter--show-password label.ds-c-label {
+    margin: 24px 0 0 0;
```

Comment:
So it looks like the original issue of the checkbox being too high has to do with the fact that `TextField`s look to inherit a clearfix. If we change this rule to just `margin-top: 0;` and then add a `className` to `TextField` that overrides the clearfix utility, these will stay in sync if a user increases the text syze. (I'll bust up an suggest commit right after this.)

Reply:
Oooooooh.

Reply:
Added a `no-clearfix` class to the `TextField` and made it essentially undo what the `TextField`'s clearfix does.  Then changed the margin to 0, and all is well!